frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.css

```diff
@@ -1,3 +1,7 @@
+body {
+  background-color: transparent;
+}
+
```

Comment:
Tried out with other themes, night mode looks as expected as far as I can tell

<img width="1291" alt="CleanShot 2023-02-16 at 15 24 30@2x" src="https://user-images.githubusercontent.com/17258145/219412220-c204d5c3-d8fa-4398-beb6-95f1c3f781cd.png">

Reply:
This looks like something that requires more thought and testing. Can this be moved to a separate PR? Don't want to block other changes